backend/wayland/output.c

```diff
@@ -253,8 +262,7 @@ static void xdg_toplevel_handle_configure(void *data,
 		return;
 	}
 	// loop over states for maximized etc?
-	wl_egl_window_resize(output->egl_window, width, height, 0, 0);
-	wlr_output_update_custom_mode(&output->wlr_output, width, height, 0);
+	output_set_custom_mode((struct wlr_output *)output, width, height, 0);
```

Comment:
Please don't cast, instead use `&output->wlr_output`

Reply:
Fixed. It looks like it's mixed throughout the file and the nearest one uses a cast.